FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift

```diff
@@ -163,3 +201,22 @@ extension ModelDownloadTask: URLSessionDownloadDelegate {
     }
   }
 }
+
+/// Extension to handle background downloading.
+extension ModelDownloadTask: UIApplicationDelegate {
```

Comment:
These methods will need to be somehow called from the app delegate (e.g. directly by the app dev or via GULAppDelegateSwizzler, etc.).